v20.1/drop-role.md

```diff
@@ -19,6 +19,8 @@ The `DROP ROLE` [statement](sql-statements.html) removes one or more SQL roles.
 
 Roles can only be dropped by super users, i.e., members of the `admin` role.
```

Comment:
This is no longer true - having CREATEROLE is sufficient.

Reply:
Ditto for drop user - DELETE privilege is not required on system.users. 
In general permissions on system.users are no longer required for these RBAC commands.